code401challenges/ArrayReverse.java

```diff
@@ -7,6 +7,12 @@ public static void main(String[] args) {
   }
 
   public static int[] reverseArray(int[] arr) {
+    for (int i = 0; i < arr.length/2; i++){
+      int temp = arr[i];
+      arr[i] = arr[arr.length -1 - i];
+      arr[arr.length -1 -i] = temp;
+
+    }
     return arr;
   }
 }
```

Comment:
If you leave an empty line at the end of your files, GitHub won't leave these red symbols on the last line